packages/formiz-core/src/store.ts

```diff
@@ -14,20 +14,21 @@ import {
 } from "@/utils/form";
 import type { GetFieldSetValueOptions, Step, Store } from "@/types";
 
-export const createStore = ({
-  formId,
-  initialStepName,
-}: { formId?: string; initialStepName?: string } = {}) =>
+export const createStore = ({}: {
+  formId?: string;
+  initialStepName?: string;
+  connected?: boolean;
```

Comment:
This can be remove ? `createStore` is use only one time with `formId` in a `unregisterStep.test.ts`

Reply:
Yes good catch, it's removed